documentation/example-file-usage/mqtt-example-file-usage.md

```diff
@@ -0,0 +1,136 @@
+
+2. Using `git` or GitHub Desktop, clone this repository into a location of your choice and then navigate to that location in Finder.
+
+3. Hold down the "Control" key and click the folder in the path bar, then choose "Open in Terminal". (If you don’t see the path bar at the bottom of the Finder window, choose View > Show Path Bar)
+
+4. Install Python3 from the [Python website.](https://www.python.org/downloads/macos/)
+
+5. Enter in the following command to install the required pip packages:
+
+   ```
+   pip3 install -r requirements.txt
+   ```
+
+   If required, enter `Y` to all installation prompts.
+
+
+
+## Extracting your Auth Token
+
+Your Auth Token is the "key to the kingdom" so-to-speak. The M5 uses this to verify who can connect to it over the internet (or LAN) to make sure it's actually you or someone you've shared your printer with (although, we haven't done any testing with shared printers so that's more of an educated guess). You're printer will refuse any connections that don't present your unique Auth Token, so let's get that out for future use.
+
+1. Navigate to wherever you cloned this repository to. Open the "examples" folder and open a terminal window there, just like in the previous section.
+
+2. Type in the following command:
+
+   ```bash
+    python3 extract-auth-token.py -a
+   ```
+
+   You should get back a long sequence of numbers and letters, that's your Auth Token. Copy that to the clipboard for use in the next section.
+
+
+
+## Connecting to your M5 over MQTT
+
+Now here comes the fun part, we have everything we need to actually connect to the printer. It uses the [MQTT standard](https://mqtt.org/), to communicate with the wider world. It is possible to use this standard coupled with this repository build a tool that can remotely monitor the printer, print files, execute movement commands, pre-heat the printer, initiate the Auto-Bed Leveling routine, and just about anything else the companion phone app can. In this example, we'll simply be monitoring the printer. We can also save the output to a log file if desired.
+
+1. Navigate to wherever you cloned this repository to. Open the "examples" folder and open a terminal window there, just like in the previous section.
+
+2. Type in the following command, but replace "YOUR_AUTH_TOKEN_HERE" with your actual Auth Token and no quotes:
+
+   ```bash
+   python3 mqtt-connect.py -r us -A "YOUR_AUTH_TOKEN_HERE"
+   ```
+
+3. **[Optional Step]** If desired, you can save the contents of the output to a log file by adding `> output.log` to the end of the command in the previous step:
+
+   ```bash
+   python3 mqtt-connect.py -r us -A "YOUR_AUTH_TOKEN_HERE" > output.log
+   ```
+
+Now you should see a bunch of data updating in your terminal, this happens in 1 second intervals. It should look something like this and will be quite a bit more colorful:
```

Comment:
`s/1 second interval/roughly once per second/`

It can sometimes be a few seconds between updates, and sometimes less than a second.